Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM cidasdpdasartip.cr.usgs.gov:8447/aqcu/rserve:latest
+
+ENV GSPLOT_VERSION_DEFAULT=0.8.1
+
+COPY inst/extdata/installPackages.R /tmp/install/installPackages.R
```

Comment:
so we dont have to download it again?

Reply:
From what I could tell this same script is used for both GSPlot and Repgen, it just reads the packages to install from the DESCRIPTION file in whatever directory it is run from. So instead of downloading this GitHub like we were before it's just pulling it directly from the Repgen source code already on the machine.